### PR TITLE
ADEN-12875 | Add Pubmatic prebid video on N&R

### PR DESCRIPTION
**Pull request by rafalgiemza**
https://fandom.atlassian.net/browse/ADEN-12875

ADEN-12875 | Add Pubmatic prebid video on N&R
- affected sites: metacritic, gamespot, giantbomb & tvguide

### CODE REVIEW

**Review by nandy-andy — DISMISSED** [review]

**Review by dmnsobczak — DISMISSED** [review]

File: src/platforms/news-and-ratings/gamespot/bidders/pubmatic.ts [hunk 1]
```diff
@@ -112,6 +112,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			sizes: [[728, 90]],
 			ids: ['4856885'],
 		},
+		video: {
+			siteId: '4923774',
```
Comment (mfarul) [inline src/platforms/news-and-ratings/gamespot/bidders/pubmatic.ts:116]: this must be ids: [...]

Comment (rafalgiemza) [inline src/platforms/news-and-ratings/gamespot/bidders/pubmatic.ts:116]:
```suggestion
			ids: ['4923774'],
```

File: src/platforms/news-and-ratings/gamespot/bidders/pubmatic.ts [hunk 2]
```diff
@@ -239,6 +242,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			sizes: [[728, 90]],
 			ids: ['4856925'],
 		},
+		video: {
+			siteId: '4923775',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/gamespot/bidders/pubmatic.ts:246]:
```suggestion
			ids: ['4923775'],
```

File: src/platforms/news-and-ratings/giantbomb/bidders/pubmatic.ts [hunk 3]
```diff
@@ -38,7 +38,11 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			],
 			ids: ['4856935', '4856936'],
 		},
+		video: {
+			siteId: '4923776',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/giantbomb/bidders/pubmatic.ts:42]:
```suggestion
			ids: ['4923776'],
```

File: src/platforms/news-and-ratings/giantbomb/bidders/pubmatic.ts [hunk 4]
```diff
@@ -80,6 +84,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			sizes: [[300, 250]],
 			ids: ['4856947'],
 		},
+		video: {
+			siteId: '4923777',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/giantbomb/bidders/pubmatic.ts:88]:
```suggestion
			ids: ['4923777'],
```

File: src/platforms/news-and-ratings/metacritic/bidders/pubmatic.ts [hunk 5]
```diff
@@ -44,6 +44,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			],
 			ids: ['4856974', '4856975'],
 		},
+		video: {
+			siteId: '4923778',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/metacritic/bidders/pubmatic.ts:48]:
```suggestion
			ids: ['4923778'],
```

File: src/platforms/news-and-ratings/tvguide/bidders/pubmatic.ts [hunk 6]
```diff
@@ -142,6 +142,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			sizes: [[300, 250]],
 			ids: ['4857049'],
 		},
+		video: {
+			siteId: '4923780',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/tvguide/bidders/pubmatic.ts:146]:
```suggestion
			ids: ['4923780'],
```

File: src/platforms/news-and-ratings/tvguide/bidders/pubmatic.ts [hunk 7]
```diff
@@ -216,6 +219,9 @@ export function getPubmaticContext(isDesktop: boolean): object {
 			sizes: [[728, 90]],
 			ids: ['4857068'],
 		},
+		video: {
+			siteId: '4923781',
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/tvguide/bidders/pubmatic.ts:223]:
```suggestion
			ids: ['4923781'],
```

**Review by mfarul — APPROVED** [review]

**Pull request merged**